### DISCUSSION TITLE
How do i disable the default dark mode while using Tailwind?

### DISCUSSION THREAD

**Discussion by emrade** [rappasoft/laravel-livewire-tables#678] [Q&A]
Hi guys, i just installed this, version `1.24` and it seems to have dark mode enable by default. I have searched the docs on how to disable that but i did not find anything. Could someone please help me?

**Comment by rappasoft** [comment]
It's not enabled by default. Something on your page or Tailwind config must be [turning it on](https://tailwindcss.com/docs/dark-mode).
